composer.json

```diff
@@ -4,9 +4,9 @@
     "type": "library",
     "license": "MIT",
     "scripts": {
-        "analyse": "./vendor/bin/phpstan analyse",
-        "bench": "./vendor/bin/phpbench run --report=redis",
-        "bench:verbose": "./vendor/bin/phpbench run --report=redis --progress=blinken"
+        "analyse": "phpstan analyse",
+        "bench": "phpbench run --report=redis",
+        "bench:verbose": "phpbench run --report=redis --progress=blinken"
```

Comment:
Can you explain this change? Why use global binaries, not the vendor ones?

Reply:
Oh noooooooo! :)

Composer is witty. Please see the purple box on https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands